apps/video/test/marionette/lib/video.js

```diff
@@ -192,6 +195,66 @@ Video.prototype = {
   },
 
   /**
+   * Taps 'pause' button and checks that the counter is not progressing.
+   * returns false if the counter has progressed.
+   */
+  pausePlayback: function() {
+    this.client.helper.waitForElement(Video.Selector.playpauseButton).click();
+    var currentTime = this.getElapsedTimeSeconds();
+    
+    // wait for 2 seconds
+    this.client.executeAsyncScript(function () 
+      {setTimeout(function(){
+        marionetteScriptFinished();
+      }, 2000);
```

Comment:
Did you notice when running the test that it's necessary to wait for a couple of seconds after clicking the button to ensure the video has been paused? In other words, have you noticed that 'getElapsedTimeSeconds' changes from line 203 to line 212?

If it does change, it's not ideal to have to wait for two seconds on each run of the test. Also, two seconds seems arbitrary. If you're up for it, I have a suggestion for possibly removing the need to wait at all. My suggestion is to move this line of code [1] to after line 1266 of video.js. And then, in 'pausePlayback', after line 202, you can wait for the 'play' button 'data-l10n-id' attribute to become 'play-button' (see [2]). Once that happens, we know 'pause' on the video element has been invoked in which case it should be safe to assume the elapsed time will not change.

[1] https://github.com/mozilla-b2g/gaia/blob/master/apps/video/js/video.js#L1257
[2] https://github.com/mozilla-b2g/gaia/blob/master/apps/video/js/video.js#L1233